### DISCUSSION TITLE
Error in GELF docs

### DISCUSSION THREAD

**Discussion by hankhank10** [datalust/seq-tickets#1121] [General]
In https://docs.datalust.co/docs/using-gelf

The docs call for installation using this command:

````
$ docker run \
    --rm \
    -it \
    -p 12201:12201/udp \
    -e SEQ_ADDRESS=https://seq.example.com:5341 \
    datalust/seq-input-gelf
````

In fact the last line should read

`-d datalust/seq-input-gelf`

This confused me for a while...

**Comment by larenelg** [comment]
Hi @hankhank10, thanks for the feedback! Recommending to run in daemon mode is a great suggestion. We'll update the docs :)
